pkg/idp/provider/github/github.go

```diff
@@ -329,10 +329,25 @@ func (g *Github) CleanCredentialsForAuthenticatedApp(config provider.AppConfig)
 	return nil
 }
 
+func (g *Github) DeleteAuthenticatedApp(config provider.AppConfig) error {
```

Comment:
In case of github, we can only open a link to the deletion page of the app since the client can not delete applications.